### DISCUSSION TITLE
openHASP 7.0 - Integration with Home Assistant

### DISCUSSION THREAD

**Discussion by DSRocker** [HASwitchPlate/openHASP#298] [General]
Hi,
Version 7.0:
integration / auto-discovery with Home Assistant doesn't work.
Could you please check it.
Thanks

**Comment by DSRocker** [comment]
Home Assistant log:
This error originated from a custom integration.

Logger: custom_components.openhasp.config_flow
Source: custom_components/openhasp/config_flow.py:75
Integration: openHASP (documentation, issues)
First occurred: 5:43:12 PM (1 occurrences)
Last logged: 5:43:12 PM

Discovery Only

**Reply by fvanroie** [reply]
You need to [uncomment 1 line](https://github.com/HASwitchPlate/openHASP-custom-component/issues/87#issuecomment-1015941574) in the Custom Component for it to work with 0.7.0.

**Reply by DSRocker** [reply]
Which file?

**Reply by fvanroie** [reply]
It is in that link:
> I found that commenting out [line #102](https://github.com/HASwitchPlate/openHASP-custom-component/blob/762e55fbe1cf8815a1d56ed2a5f55e94d40b79be/custom_components/openhasp/config_flow.py#L102) will let you discover the `0.7` devices.

**Reply by DSRocker** [reply]
oh ok... I was looking for the file in openHASP and not in the Custom Component ;-)
Thanks; it seems to work.
Without integration you can use it too; the device is shown like a mqtt.
